app/javascript/lib/sentry-logging.js

```diff
@@ -54,12 +89,17 @@ function shouldLog() {
  *  Initialize Sentry to enable logging JS errors to Sentry
  */
 export function setupSentry() {
+  if (isSuppressedEnv) {
+    const reason = 'in an unlogged environment'
+    printSuppression(reason)
```

Comment:
Couldn't you just call `printSuppression()` (without the if statement here) since it has the same logic to check if it's a `isSuppressedEnv` ?

Reply:
Good catch!  Will do.